frontend/src/lib/components/LemonBubble/LemonBubble.scss

```diff
@@ -1,93 +1,94 @@
 @import '~/vars';
 
-$lemon-counter-small: 1.2rem;
-$lemon-counter-font-small: 0.6rem;
-$sf-small: 0.8;
-$sf-large: 1.2;
-$position-offset: 0.5rem;
-
 .LemonBubble {
+    --lemon-counter-small: 1.2rem;
+    --lemon-counter-font-small: 0.6rem;
```

Comment:
One more thing: I'm a bit confused why this is `-small` if `--scale-small` is a separate multiplier

Reply:
Good catch. Should be "size"